website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -407,3 +407,7 @@ This table shows v1beta1 metrics that were dropped for v1:
 | NodeClaim   | karpenter_nodeclaims_drifted |
 | Provisioner | karpenter_provisioner_scheduling_duration_seconds |
 | Interruption | karpenter_interruption_actions_performed |
+
+{{% alert title="Note" color="warning" %}}
+Karpenter now waits for the underlying instance to be completely terminated before deleting a node and orchestrates this by emitting `NodeClaimNotFoundError`. With this change we expect to see an increase in the `NodeClaimNotFoundError`. Customers can filter out this error in order to get accurate values for `karpenter_cloudprovider_errors_total` metric.
```

Comment:
```suggestion
Karpenter now waits for the underlying instance to be completely terminated before deleting a node. As a result we emit `NodeClaimNotFoundError` on node termination for  `karpenter_cloudprovider_errors_total` . Filter by label to not consider these events. 
```

Reply:
I actually prefer some of the original wording -- though I would say that I agree that we should probably provide an example of the actual prometheus filter that we would use here to make this more clear to users